src/app/components/JumpTo/index.stories.tsx

```diff
@@ -17,11 +25,26 @@ export default {
   },
 };
 
+export const ExampleWithRelatedContentLink = () => {
+  const jumpToBlock =
+    pidginArticleFixtureWithJumpToBlock.data.article.content.model.blocks.find(
+      block => block.type === 'jumpTo',
+    );
+
+  const jumpToHeadings = jumpToBlock?.model.jumpToHeadings ?? [];
+
+  return <Component jumpToHeadings={jumpToHeadings} showRelatedContentLink />;
+};
+
 export const Example = () => {
-  const jumpToHeadings =
+  const jumpToBlock =
     pidginArticleFixtureWithJumpToBlock.data.article.content.model.blocks.find(
       block => block.type === 'jumpTo',
-    )?.model.jumpToHeadings;
+    );
+
+  const jumpToHeadings = jumpToBlock?.model.jumpToHeadings ?? [];
 
-  return <Component jumpToHeadings={jumpToHeadings} />;
+  return (
+    <Component jumpToHeadings={jumpToHeadings} showRelatedContentLink={false} />
```

Comment:
You don't technically need this, as it is defaulted to false
```suggestion
    <Component jumpToHeadings={jumpToHeadings} />
```